src/xcode/ENA/ENA/Source/Developer Menu/Features/DMSubmissionStateViewController.swift

```diff
@@ -189,7 +189,8 @@ private extension Client {
 	typealias AvailableDaysAndHoursCompletion = (DaysAndHours) -> Void
 
 	private func availableDaysAndHours(
-		completion completeWith: @escaping AvailableDaysAndHoursCompletion
+		completion completeWith: @escaping AvailableDaysAndHoursCompletion,
+		wifiClient: WifiOnlyHTTPClient
```

Comment:
Seems kind of unusual not to have the completion at the end of the parameter list

Reply:
Right. Didn't noticed. Moved it at the end.